### ISSUE TITLE
DTS 2.0.0-rc3 not able to handle proprietary EC (eg. after a failed EC sync update)

### ISSUE THREAD

**Issue by akoskoronka** [Dasharo/dasharo-issues#966]
### Component

Dasharo Tools Suite, EC firmware

### Device

NovaCustom V54 14th Gen

### Dasharo version

v0.9.0

### Dasharo Tools Suite version

2.0.0-rc3

### Brief summary

dasharo_ectool cannot connect to the proprietary EC, updater script also fails since it cannot sense AC charger required for flashing.

### How reproducible

100%

### How to reproduce

Boot DTS 2.0.0-rc3 on device running proprietary EC and try to update firmware (downgrade to any upgradable), or run `dasharo_ectool info` from shell.

### Expected behavior

Proprietary EC should be detected during update and updating should be offered.

### Actual behavior

Updater script stuck with "Warning: AC adapter must be connected before performing firmware update. Please connect the AC adapter and press 'C' to continue, or 'Q' to quit.". Running `dasharo_ectool info` in shell results in "failed to connect to EC: WouldBlock" error message.

### Additional context

ecfix.sh was provided as a workaround.

**Comment by miczyg1** [comment]
@akoskoronka `dasharo_ectool` is not supposed to work with proprietary EC if I am correct. @mkopec please confirm.

To flash the proprietary EC, the flashrom with ite_ec programmer is the way to go.

**Comment by mkopec** [comment]
> dasharo_ectool is not supposed to work with proprietary EC if I am correct. 

Correct, when proprietary EC is installed, flashrom ite_ec must be used.
